src/Compilers/VisualBasic/Portable/Lowering/MethodToClassRewriter/MethodToClassRewriter.vb

```diff
@@ -57,12 +57,14 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
         Protected ReadOnly Diagnostics As DiagnosticBag
         Protected ReadOnly SlotAllocatorOpt As VariableSlotAllocator
+        Protected ReadOnly PreserveOriginalLocals As Boolean
```

Comment:
Consider adding a comment explaining what does this mean and when it might be true.